src/app/checklist/project-exists.guard.ts

```diff
@@ -20,7 +20,7 @@ export class ProjectExistsGuard implements CanActivate {
       select(ProjectsSelectors.getProjectById(projectId)),
       switchMap(project => {
         if (!project) {
-          this.router.navigate(['/projects']);
+          void this.router.navigate(['/projects']);
```

Comment:
Why do we need the `void` here?

Reply:
We do not **need** it to make it run.
Some IDE warns about the Promise returned is ignored without it.
I can remove it if you want.

Reply:
Nah we can keep it 👍